### PR TITLE
Add bionic image

### PR DESCRIPTION
**Pull request by Nirothipan**
$subject

### CODE REVIEW

File: bionic.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+FROM ubuntu:bionic
+
+RUN \
+    apt-get update \
+    && apt-get install -y wget \
+    && apt-get install -y curl
+
+RUN \
+    apt-get -q update && apt-get install -y openssh-server software-properties-common git ant curl zip unzip xvfb dbus-x11 ttf-ancient-fonts \
```
Comment (chirangaalwis) [inline bionic.Dockerfile]: @Nirothipan shall we please break the `apt-get update` and `apt-get install` commands into multiple lines?

And please move package names into multiple lines in alphabetical order for clarity (see [example](https://github.com/wso2/docker-apim/blob/v3.1.0.3/dockerfiles/ubuntu/apim/Dockerfile#L58)).

Comment (chirangaalwis) [reply bionic.Dockerfile]: Please check for any other applicable instances within the PR, if any.

**Review by chirangaalwis — CHANGES_REQUESTED** [review]

File: bionic.Dockerfile (hunk 1 above)
Comment (Nirothipan) [reply bionic.Dockerfile]: @chirangaalwis Thanks for the review. Fixed.

File: bionic.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+# limitations under the License
+
+# ------------------------------------------------------------------------
+
+FROM ubuntu:bionic
+
+RUN \
+    apt-get -q update \
+    && apt-get install -y \
+       openssh-server \
```
Comment (chirangaalwis) [inline bionic.Dockerfile]: @Nirothipan minor change. `openssh-server` is not in order.

**Review by chirangaalwis — CHANGES_REQUESTED** [review]

File: bionic.Dockerfile (hunk 2 above)
Comment (Nirothipan) [reply bionic.Dockerfile]: Thanks . Fixed it .

**Review by chirangaalwis — APPROVED** [review]

**Pull request merged**